docs/integrations/web-servers/opentelemetry/apache-opentelemetry.md

```diff
@@ -0,0 +1,297 @@
+
+Configure Apache to log to a local files. Before you can configure Sumo Logic to ingest logs, you must configure the logging of access logs and error logs via the instructions described in their [documentation](https://httpd.apache.org/docs/2.4/logs.html).
+
+To configure the Apache log files, locate your local `httpd.conf` configuration file in the Apache directory. After determining the location of the conf file, modify the httpd.conf configuration file logging parameters if required.
+
+For access logs, the following directive is to be noted:
+-   CustomLog: access log file path and format (standard common and combined)
+
+For error logs, following directives are to be noted:
+-   ErrorLog: error log file path
+-   LogLevel: to control the number of messages logged to the `error_log`
+
+## Collecting Logs, Metrics & Installing App for Apache
+
+Here are the steps for Collecting Logs, metric and installing the app:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+If you want to use an existing OpenTelemetry Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs to get monitored. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Apache-OpenTelemetry/Apache-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step, we will be configuring the yaml required for Apache Collection.
+
+Below are the input required:
+
+-   **Endpoint** - (default: http://localhost:8080/server-status?auto): The URL of the httpd status endpoint
+-   **Access File log Path** - Enter the path to the Access log file for your mysql instance.
+-   **Error file log path** - Enter the path to the error log file for your mysql instance.
+-   Fields - **webengine.cluster.name**
+You can add any custom fields which you want to tag along with the data ingested in sumo.
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Apache-OpenTelemetry/Apache-YAML.png')} alt="YAML" />
+
+### Step 3: Sending logs and metric to Sumo
+
+Once you download the yaml file, please follow the below steps based on your platform.
+
+<Tabs
+  className="unique-tabs"
+  defaultValue="Linux"
+  values={[
+    {label: 'Linux', value: 'Linux'},
+    {label: 'Windows', value: 'Windows'},
+    {label: 'macOS', value: 'macOS'},
```

Comment:
```suggestion
    {label: 'Mac', value: 'Mac'},
```

Reply:
I'm going to stet these to maintain consistency with the rest of our docs, which all use macOS. Definitely open to discussing at a team meeting!